src/lib/Pagination/Mapper/AbstractPagerContentToDataMapper.php

```diff
@@ -88,19 +88,17 @@ protected function isContentIsUser(Content $content): bool
     }
 
     /**
-     * @param \eZ\Publish\API\Repository\Values\Content\ContentInfo $contentInfo
+     * @param \eZ\Publish\API\Repository\Values\Content\VersionInfo $versionInfo
      *
      * @return \eZ\Publish\API\Repository\Values\User\User|null
      */
-    protected function getContributor(ContentInfo $contentInfo): ?User
+    protected function getVersionContributor(VersionInfo $versionInfo): ?User
     {
-        $userExists = (new UserExists($this->userService))->isSatisfiedBy($contentInfo->ownerId);
-
-        if (false === $userExists) {
+        try {
```

Comment:
Any reason why you did not reuse `Specification` just with different ID provided?

Reply:
@ViniTou This method called for each row, so this will save us a few `userService->loadUser()` calls, which will make the dashboard page a bit faster.
IMHO, `UserExists->isSatisfiedBy` is overcomplicating the simple thing, user service is doing this job quite well already. Probably, it is the reason why it's not used commonly through the code.

Let me know if these few extra calls are not a big deal and I'll change it to use `UserExists`.
Thanks.

Reply:
True, I didnt notice the last removed call, fine with me then.